Sources/WebSocketKit/WebSocketClient.swift

```diff
@@ -124,21 +143,43 @@ public final class WebSocketClient: Sendable {
                         upgradeRequestHeaders.add(contentsOf: proxyHeaders)
                     }
                 }
+                print("websocket-client: config is \(self.configuration)")
 
+                if upgradeRequestHeaders.contains(name: "sec-websocket-extensions") {
+                
+                    print("websocket-client: upgrade request headers are \(upgradeRequestHeaders)")
```

Comment:
This is a great use case for logging again. Though I'm not sure we'll need request headers in our logs. What's the reasoning here?